backend/src/main/java/com/redhat/cloud/notifications/auth/kessel/KesselAssets.java

```diff
@@ -0,0 +1,154 @@
+package com.redhat.cloud.notifications.auth.kessel;
+
+import com.redhat.cloud.notifications.routers.SecurityContextUtil;
+import io.grpc.StatusRuntimeException;
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.Tags;
+import io.micrometer.core.instrument.Timer;
+import io.quarkus.logging.Log;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+import jakarta.ws.rs.core.SecurityContext;
+import org.project_kessel.api.inventory.v1beta1.CreateNotificationsIntegrationRequest;
+import org.project_kessel.api.inventory.v1beta1.CreateNotificationsIntegrationResponse;
+import org.project_kessel.api.inventory.v1beta1.DeleteNotificationsIntegrationRequest;
+import org.project_kessel.api.inventory.v1beta1.DeleteNotificationsIntegrationResponse;
+import org.project_kessel.api.inventory.v1beta1.Metadata;
+import org.project_kessel.api.inventory.v1beta1.NotificationsIntegration;
+import org.project_kessel.api.inventory.v1beta1.ReporterData;
+import org.project_kessel.client.NotificationsIntegrationClient;
+
+@ApplicationScoped
+public class KesselAssets {
+    /**
+     * Represents the timer's name for checking how much time it takes to
+     * perform operations in Kessel for the "integration" resource.
+     */
+    private static final String KESSEL_METRICS_INVENTORY_INTEGRATION_TIMER_NAME = "notifications.kessel.inventory.resources";
+    /**
+     * The resource format we need to use when deleting resources from the
+     * Kessel inventory.
+     */
+    public static final String RESOURCE_FORMAT = "reporter_data.%s:reporter_data.%s";
+
+    @Inject
+    MeterRegistry meterRegistry;
+
+    @Inject
+    NotificationsIntegrationClient notificationsIntegrationClient;
+
+    /**
+     * Creates an integration in Kessel's inventory.
+     * @param securityContext the security context which holds the subject of
+     *                        the operation.
+     * @param workspaceId the identifier for the workspace the integration will
+     *                    be created on.
+     * @param integrationId the identifier of the integration we created in
+     *                      our side.
+     */
+    public void createIntegration(final SecurityContext securityContext, final String workspaceId, final String integrationId) {
```

Comment:
Unless I missed something, `securityContext` is only used for logging purposes in this method and the other one. What is the point of logging the identity if it's not part of the request sent to Kessel?

Reply:
This is a great question. Actually I think we are going to be creating the resources on the user's behalf, but I am unsure if we will need to specify those details in the future. Let me ask the Kessel team.

Reply:
Let's make the changes (if any) in a follow-up PR.